public/app/views/shared/_note_content.html.erb

```diff
@@ -0,0 +1,34 @@
+<%
+  content_length = content.length
+  max_chars = AppConfig[:pui_readmore_max_characters]
+%>
+
+<div class="d-flex flex-column align-items-start wrap-anywhere note-content">
+  <% unless is_inherited.blank? %>
+    <%= inheritance(is_inherited).html_safe %>
+  <% end %>
+
+  <% if (calling_partial != 'record_innards' || content_length <= max_chars) %>
+    <%= content.html_safe %>
+  <% else %>
+    <div class="position-relative mb-3 d-flex flex-column align-items-start" data-js="readmore">
+      <input
+        type="checkbox"
+        id="<%= type %>_readmore"
```

Comment:
The id attribute must be unique within the HTML document.

It fails for more than one Biographical / Historical (note_multipart). If you click on the Read More button for the second note of the same type, then it always expands the first one.

Possibly Make the id unique, either by passing an incremental value, preferably generate a unique id with UUID with SecureRandom.uuid?